src/devtools/views/Components/SelectedElement.js

```diff
@@ -170,11 +183,21 @@ function InspectedElementView({
         overrideValueFn={overridePropsFn}
         showWhenEmpty
       />
-      <InspectedElementTree
-        label="state"
-        data={state}
-        overrideValueFn={overrideStateFn}
-      />
+      {type === ElementTypeSuspense ? (
+        <InspectedElementTree
+          label="suspense"
+          data={{
+            [IS_SUSPENDED]: state !== null,
+          }}
+          overrideValueFn={overrideSuspenseFn}
+        />
+      ) : (
+        <InspectedElementTree
+          label="state"
+          data={state}
+          overrideValueFn={overrideStateFn}
+        />
+      )}
```

Comment:
I hope we don't end up needing more branching like this 😅

Reply:
Can refactor if we do. :-) I haven’t seen other cases but they might come up if we add `memoizedState` to other types.